src/modules/sub-items/components/table-view/table.view.columns.toggler.js

```diff
@@ -28,6 +32,16 @@ export default class TableViewColumnsTogglerComponent extends Component {
         }));
     }
 
+    componentDidUpdate() {
+        const { isOpen, panelHeight } = this.state;
+
+        if (isOpen && panelHeight === null) {
```

Comment:
I'm making explicit `panelHeight === null` check in case of `this._refPanel.offsetHeight` returning `0`.